app/views/groups/_member.html.erb

```diff
@@ -0,0 +1,21 @@
+<li itemscope itemtype="http://schema.org/Code">
+  <div class="header">
+    <div class="header-content">
+      <%= link_to member.user, rel: 'member', itemprop: 'username' do %><%= member.user.username %><% end %>
+      <div class="remove">
+        <% if admin_member?(current_user, member.group) %>
+          <% if member.admin? %>
+            <% if admin_members(member.group).count > 1 %>
```

Comment:
Also noticed `@admin_members` in the `GroupsController` that can be used at least for the count instead of a query to the database through `admin_members` method from the helper.

```
admin_member?(current_user, member.group)
``` 
... might be able to be replaced with ...

```
@admin_members.map(&:id).include?(current_user.id)
```

... maybe? To avoid more trips to the database? Maybe in the helper?